evalml/demos/diabetes.py

```diff
@@ -13,12 +14,17 @@ def load_diabetes():
         (pd.Dataframe, pd.Series): X and y
     """
     filename = (
-        "https://api.featurelabs.com/datasets/diabetes.csv?library=evalml&version="
+        "https://api.featurelabs.com/datasets/diabetes-2022-06-27.csv?library=evalml&version="
         + evalml.__version__
     )
     X, y = load_data(filename, index=None, target="target")
     y.name = None
-
+    X = X.astype(float)
+    y = y.astype(float)
+    numpy_of_X = X.to_numpy()
+    numpy_of_X = scale(numpy_of_X, copy=False)
+    numpy_of_X /= numpy_of_X.shape[0] ** 0.5
```

Comment:
To be honest, I simply copied what sklearn does inside their own code and don't really know why the regularization does this. 

While I don't understand the math or reasoning behind the division, from my understanding of the methods, load_diabetes from evalml before simply loaded the dataset in a similar manner to how sklearn did it. When scikit learn updated to 1.1.1, inside of sklearn's own code, they added this regularization to the diabetes dataset before they returned, rather than just simply returning the dataset like they had previously.

This happened because now the diabetes.csv file is no longer pre-regularized, and is actually the whole numbers (so the values are now numbers like 116 or 81 rather than stuff like 0.00116 or 0.00081). Therefore, to pass the test, I thought to just imitate what sklearn now does with its diabetes dataset.

Here's a screenshot of the load_diabetes() method from scikit learn which shows the difference between v1.0.2 and v1.1.1.

scikit v1.0.2:
<img width="833" alt="Screen Shot 2022-07-05 at 9 00 49 AM" src="https://user-images.githubusercontent.com/56745347/177369849-a004b9e0-e682-4957-b490-757520575962.png">

scikit v1.1.1
<img width="848" alt="Screen Shot 2022-07-05 at 9 01 17 AM" src="https://user-images.githubusercontent.com/56745347/177370073-2bd2f7d2-ae11-4515-baf7-c8afbc947bf5.png">

They also added this note to the sklearn's `load_diabetes()` method docstring:

<img width="677" alt="Screen Shot 2022-07-05 at 9 05 01 AM" src="https://user-images.githubusercontent.com/56745347/177370492-13a57405-fc50-4f69-a51f-677149fd062c.png">

And they also changed the function parameters so that default, scaled = True: 
<img width="649" alt="Screen Shot 2022-07-05 at 9 12 09 AM" src="https://user-images.githubusercontent.com/56745347/177371536-024e6d89-e1bc-4cb4-af5c-e50e30637b46.png">

Upon looking at the screenshot of the docstring, I guess they wanted to scale the numbers by the standard deviation * sqrt(n_samples). Dunno why, but by doing that the numbers that diabetes.csv output is similar (but not quite the same) as the numbers that diabetes.csv had previously.